cpp/src/slice2cpp/Gen.cpp

```diff
@@ -72,18 +72,19 @@ namespace
         }
     }
 
-    string getDeprecateSymbol(const ContainedPtr& p1, const ContainedPtr& p2)
+    string getDeprecateSymbol(const ContainedPtr& p1)
     {
-        string deprecateMetadata, deprecateSymbol;
-        if (p1->findMetaData("deprecate", deprecateMetadata) ||
-            (p2 != 0 && p2->findMetaData("deprecate", deprecateMetadata)))
+        string deprecateSymbol;
```

Comment:
I'd use "deprecated" instead of "deprecate".

Reply:
Also fixed!